database/inteldb/inteldb_test.go

```diff
@@ -1,122 +0,0 @@
-package inteldb
```

Comment:
This test no longer reflected the code within in the project. It was broken :( A new test should be written. We can grab the code out of the repo history if there is anything to reuse.